medicines/web/src/services/azure-search.ts

```diff
@@ -98,6 +57,10 @@ const buildSearchUrl = (
   url.searchParams.append('search', query);
   url.searchParams.append('scoringProfile', searchScoringProfile as string);
   url.searchParams.append('searchMode', 'all');
+  const f = createFilter(filters);
```

Comment:
`filter`

Reply:
extracted this out so I can do some descriptive naming in the function, never liked how big this method was getting